components/templates/ReviewTemplate.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { StyleSheet, useColorScheme } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { ScrollView, Text, View, XStack, YStack } from 'tamagui'
+
+import ChooseImage from '~/components/atoms/ChooseImage'
+import InputReview from '~/components/atoms/InputReview'
+import { PositiveButton } from '~/components/atoms/PositiveButton'
+import LinearGradientBackground from '~/components/molecules/LinearGradientBackground'
+import PickRating from '~/components/molecules/PickRating'
+import getColors from '~/constants/Colors'
+import { useAppFonts } from '~/hooks/useAppFonts'
+import useTranslation from '~/hooks/useTranslation'
+
+const ReviewTemplate = (): React.ReactElement => {
+  const colors = getColors(useColorScheme())
+  const { fonts } = useAppFonts()
+  const { t } = useTranslation()
+
```

Comment:
Consider the extra handling (Error Handling) for the Input and the image selection process, such as warning when entering the data invalid or when it is impossible to download